pom.xml

```diff
@@ -549,20 +549,20 @@
         <!--Maven Plugin Version-->
         <maven.compiler.plugin.version>2.3.1</maven.compiler.plugin.version>
         <maven.bundle.plugin.version>2.4.0</maven.bundle.plugin.version>
-        <jackson-jaxrs-json-provider.version>2.9.9</jackson-jaxrs-json-provider.version>
-        <jackson-databind.version>2.9.9.3</jackson-databind.version>
-        <jackson-dataformat-xml.version>2.9.9</jackson-dataformat-xml.version>
-        <cxf-bundle.version>3.2.8</cxf-bundle.version>
+        <jackson-jaxrs-json-provider.version>2.10.5</jackson-jaxrs-json-provider.version>
+        <jackson-databind.version>2.10.5</jackson-databind.version>
+        <jackson-dataformat-xml.version>2.10.5</jackson-dataformat-xml.version>
+        <cxf-bundle.version>3.3.7</cxf-bundle.version>
         <olingo-odata2.version>1.1.0</olingo-odata2.version>
         <jackson.version>1.9.13</jackson.version>
         <spring-web.version>5.1.1.RELEASE</spring-web.version>
-        <swagger-jaxrs.version>1.5.20</swagger-jaxrs.version>
+        <swagger-jaxrs.version>1.6.2</swagger-jaxrs.version>
         <javax.ws.rs-api.version>2.1.1</javax.ws.rs-api.version>
         <junit.version>4.12</junit.version>
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
-        <identity.user.api.version>1.1.6</identity.user.api.version>
-        <identity.server.api.version>1.0.153</identity.server.api.version>
+        <identity.user.api.version>1.1.7-SNAPSHOT</identity.user.api.version>
+        <identity.server.api.version>1.0.154-SNAPSHOT</identity.server.api.version>
```

Comment:
After merging [1] and [2] the released versions should be added. Therefore don't merge this PR until [1] and [2] is merged.

[1] https://github.com/wso2/identity-api-server/pull/198
[2] https://github.com/wso2/identity-api-user/pull/109